adapters/http-base/src/main/java/org/eclipse/hono/adapter/http/impl/VertxBasedHttpProtocolAdapter.java

```diff
@@ -141,6 +146,7 @@ private void addTelemetryApiRoutes(final Router router, final Handler<RoutingCon
                         .allowedHeader(HttpHeaders.CONTENT_TYPE.toString())
                         .exposedHeader(Constants.HEADER_COMMAND)
                         .exposedHeader(Constants.HEADER_COMMAND_REQUEST_ID))
+                .handler(getBodyHandler())
                 .setName("/telemetry (CORS)");
```

Comment:
Adding the BodyHandler to the Route with name "/telemetry (CORS)" means that a too large request will appear in the tracing UI with a span named "/telemetry (CORS)".
That would be misleading. It would be better to add the BodyHandler to other route(s).

When testing the above, I also saw that the span for the too large request didn't get an error tag.